static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -12,6 +12,10 @@ const RECURRING_CONTRIBUTOR_COOKIE = 'gu_recurring_contributor';
 const AD_FREE_USER_COOKIE = 'GU_AF1';
```

Comment:
maybe rename `RECURRING_CONTRIBUTOR_COOKIE` to `SIGNED_IN_RECURRING_CONTRIBUTOR_COOKIE`